src/pages/home/index.js

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import '../../App.css'
+import './home.css'
+import ItemTemplate from '../../components/item-template'
+import ItemList from './items-list.js'
+import {ItemContext} from '../../context/items'
 
 
-function App() {
+
+function Home(props) {
+
+  const item = useContext(ItemContext)
+  
+
   return (
-    <div style={{ textAlign: 'center', display: 'flex', justifyContent: 'center', height: '100vh', flexDirection: 'column' }}>
-       Home page
+    <div className='Background'>
+      <h1 className='Title'>Welcome to the Store!</h1>
+      <div className='Item-container'>
+        {ItemList.map((items,) => <ItemTemplate image={items.image} title={items.title} price={items.price} onClick={() =>(props.history.push(items.location), item.updateItem(items.location))}/>)}
```

Comment:
rather than having the comma in your onClick you should make it a multi line function using {}

I think rather than pushing the location we should have an id for each item and then store that. the idea is the same but it makes it a lot clearer.

Once you done that you'll want to use that id when you push to the correct route so rather than `items.location` it would be something like `props.history.push(`/items/${items.id}`)